pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -393,10 +395,17 @@ private void addKitRequest(Connection conn, String kitTypeName, KitRequestSettin
                     errorMessage += "collaboratorSampleId was too long ";
                 }
             }
-            KitRequestShipping.writeRequest(ddpInstance.getDdpInstanceId(), shippingId, kitTypeId, kit.getParticipantId().trim(),
-                    collaboratorParticipantId, collaboratorSampleId, userId, addressId, errorMessage, kit.getExternalOrderNumber(), false,
-                    uploadReason, ddpInstance);
-            kit.setDdpLabel(shippingId);
+            if(ddpInstance.getDdpInstanceId().equals("30")){
```

Comment:
we should keep it flexible so that if other studies would join that way we don't need to make changes all the time

Reply:
I changed it to `if(ddpInstance.getParticipantIndexES() == null)` to try and make it more flexible. Does that work?